src/common/services/request/query/edit-play.js

```diff
@@ -0,0 +1,27 @@
+export const updatePlayInfo = (obj) => {
+    return {
+      display: "Update Play Info",
+      name: "update_play",
+      function: "update_play",
+      write: true,
+      where: {
+        clause: {
+          operator: "and",
+          conditions: [
+            {
+              field: "play_id",
+              operator: "eq",
+              value: obj.play_id,
+            },
+            {
+              field: "user_id",
```

Comment:
play_id is unique across the platform then , do you need user_id also to be in where

Reply:
i think yes bcz its also about who (user_id) is requesting to edit which play. and more good to put token with the request also otherwise any request with valid play_id and user_id will make changes into our backend without proving to be owner?